src/Messages/Messages.js

```diff
@@ -194,21 +194,46 @@ export function Messages({ fromUserId }) {
-                    className={styles.messagesInput}
-                  />
-                  <svg
-                    aria-label="Add Photo or Video"
-                    fill="#262626"
-                    height="24"
-                    role="img"
-                    viewBox="0 0 48 48"
-                    width="24"
+                  <form
+                    className={styles.inputForm}
+                    onSubmit={(event) => {
+                      event.preventDefault();
+                      const message = event.target.messageBody.value;
+                      submitMessage(userIndex, message);
+                    }}
                   >
-                    <path d="M38.5 0h-29C4.3 0 0 4.3 0 9.5v29C0 43.7 4.3 48 9.5 48h29c5.2 0 9.5-4.3 9.5-9.5v-29C48 4.3 43.7 0 38.5 0zM45 38.5c0 3.6-2.9 6.5-6.5 6.5h-29c-3.3 0-6-2.5-6.4-5.6l8.3-8.3c.1-.1.3-.2.4-.2.1 0 .2 0 .4.2l6.3 6.3c1.4 1.4 3.6 1.4 5 0L35.9 25c.2-.2.6-.2.8 0l8.3 8.3v5.2zm0-9.4l-6.2-6.2c-1.3-1.3-3.7-1.3-5 0L21.3 35.3c-.1.1-.3.2-.4.2-.1 0-.2 0-.4-.2L14.2 29c-1.3-1.3-3.7-1.3-5 0L3 35.2V9.5C3 5.9 5.9 3 9.5 3h29C42.1 3 45 5.9 45 9.5v19.6zM11.8 8.2c-1.9 0-3.5 1.6-3.5 3.5s1.6 3.5 3.5 3.5 3.5-1.6 3.5-3.5-1.6-3.5-3.5-3.5z"></path>
-                  </svg>
+                    <input
+                      name="messageBody"
+                      type="text"
+                      placeholder="Message..."
+                      className={styles.messagesInput}
+                      onChange={(e) => {
+                        setMessageFieldText(e.target.value);
+                      }}
+                      value={messageFieldText}
+                    />
+
+                    <button
+                      className={cx(styles.sendButton, {
+                        [styles.isText]: messageFieldText !== '',
+                      })}
+                    >
+                      {messageFieldText !== '' ? (
+                        'Send'
+                      ) : (
+                        <svg
+                          aria-label="Add Photo or Video"
+                          fill="#262626"
+                          height="24"
+                          role="img"
+                          viewBox="0 0 48 48"
+                          width="24"
+                        >
+                          <path d="M38.5 0h-29C4.3 0 0 4.3 0 9.5v29C0 43.7 4.3 48 9.5 48h29c5.2 0 9.5-4.3 9.5-9.5v-29C48 4.3 43.7 0 38.5 0zM45 38.5c0 3.6-2.9 6.5-6.5 6.5h-29c-3.3 0-6-2.5-6.4-5.6l8.3-8.3c.1-.1.3-.2.4-.2.1 0 .2 0 .4.2l6.3 6.3c1.4 1.4 3.6 1.4 5 0L35.9 25c.2-.2.6-.2.8 0l8.3 8.3v5.2zm0-9.4l-6.2-6.2c-1.3-1.3-3.7-1.3-5 0L21.3 35.3c-.1.1-.3.2-.4.2-.1 0-.2 0-.4-.2L14.2 29c-1.3-1.3-3.7-1.3-5 0L3 35.2V9.5C3 5.9 5.9 3 9.5 3h29C42.1 3 45 5.9 45 9.5v19.6zM11.8 8.2c-1.9 0-3.5 1.6-3.5 3.5s1.6 3.5 3.5 3.5 3.5-1.6 3.5-3.5-1.6-3.5-3.5-3.5z"></path>
+                        </svg>
+                      )}
+                    </button>
+                  </form>
```

Comment:
You can leave these changes in, not a big deal, but they're not part of the PR. If you want to keep them, please change the PR to be "Improve messages implementation" and list the improvements you made.

Reply:
Okay